### PR TITLE
chore(ci): CI with GitHub Actions

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

We experiencing several problems with Travis CI. It is slow and does not allow to restart builds.

## What was done?

- Introduce Github Action workflows for running tests and releasing.
- Delete Travis scripts
- Update badges
- Fix invalid readme syntax

## How Has This Been Tested?

With Github Actions

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by shumkov — APPROVED** [review]
👍

**Review by shumkov — CHANGES_REQUESTED** [review]
It seems functional tests require local network.

**Comment by strophy** [comment]
I tried starting a local network both on my local machine and [in CI](https://github.com/dashevo/js-dpns-client/pull/12/checks?check_run_id=2440559860), but always get this error:
```
  0 passing (27s)
  2 failing

  1) DPNS client
       "before each" hook for "should be able to register and resolve a domain":
     Error: connect ECONNREFUSED 127.0.0.1:55582
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1144:16)

  2) DPNS client
       "after each" hook for "should be able to register and resolve a domain":
     TypeError: Cannot read property 'remove' of undefined
      at Context.<anonymous> (test/functional/dpnsClient.spec.js:275:24)
      at processImmediate (internal/timers.js:461:21)
```

What does it expect to find on this port, and where is the port defined?

**Comment by shumkov** [comment]
@strophy My bad. We don't need the local network here. The functional tests in this repo using obsolete https://github.com/dashevo/js-dp-services-ctl to start DAPI instances.

I would just skip functional tests for now. To do that, you need to replace `describe` to `describe.skip` in this line https://github.com/dashevo/js-dpns-client/blob/v0.2-dev/test/functional/dpnsClient.spec.js#L108

File: .github/workflows/.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#DRIVE_BRANCH=
```
Comment (shumkov) [inline .github/workflows/.env]: We don’t need it since we are not using mn bootsteap

File: .github/workflows/test_and_release.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  pull_request:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+-dev
+
+jobs:
+  test:
+    name: Run DPNS Client tests
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
```
Comment (shumkov) [inline .github/workflows/test_and_release.yml]: 10?

**Review by shumkov — CHANGES_REQUESTED** [review]

**Comment by strophy** [comment]
OK, I made the changes. I wonder if it is possible to include the grpc build in the cache? Currently it says this:
```
node-pre-gyp WARN Pre-built binaries not installable for grpc@1.24.2 and node@12.22.1 (node-v72 ABI, glibc) (falling back to source compile with node-gyp) 
```

Also I just added [new CI](https://github.com/dashevo/js-dp-services-ctl/pull/132) to that obsolete `js-dp-services-ctl` repo, should we archive it instead?

**Comment by shumkov** [comment]
Yeah, you can use the node-gyp-cache npm package as we are doing in Drive and DAPI.

We will archive dp-services-crl when we remove it from everywhere.

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**